<launch>
    <param name="/bsc/max_dist" type="double" value="15.0"/>
    <param name="/bsc/max_cmd" type="double" value="3.0"/>
    <param name="/bsc/start_x" type="double" value="11.5"/>
    <param name="/bsc/start_y" type="double" value="0.65"/>
    <param name="/bsc/stary_z" type="double" value="0.01"/>
    <param name="/bsc/start_w" type="double" value="0.99"/>
    <param name="/bsc/user_delay" type="boolean" value="false"/>
    <param name="/bsc/delay" type="double" value="1.0"/>
    <param name="/bsc/user_name" type="string" value="Velin"/>
    <param name="/bsc/test_type" type="string" value="BaselineBSC"/>
    <param name="/bsc/goal_pos" type="boolean" value="true"/>

    <node name="bsc_main" pkg="bsc_turtle" type="bsc_main.py"/>
    <node name="bsc_data" pkg="bsc_turtle" type="expVars.py"/>
</launch>